src/extraFiles/rpi-usb-ether-gadget.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# check if the script is run as root
+if [ "$EUID" -ne 0 ]; then
+    echo -e "\e[31mPlease run this script as root\e[0m"
+    exit
+fi
+
+# check if the script is run on a Raspberry Pi
+if ! grep -q "Raspberry Pi" /proc/device-tree/model; then
+    echo -e "\e[31mThis script is for Raspberry Pi only\e[0m"
+    exit
+fi
+
+FORCE_ON=false
+FORCE_OFF=false
```

Comment:
Might be more sensible to have a single `TURN_ON` variable, rather than separate `FORCE_ON` and `FORCE_OFF` variables?

Reply:
fixed